src/Private/Version.ps1

```diff
@@ -0,0 +1,127 @@
+<#
+.SYNOPSIS
+Retrieves the PowerShell module manifest object for the specified module.
+
+.DESCRIPTION
+This function constructs the path to a PowerShell module manifest file (.psd1) located in the parent directory of the script root. It then imports the module manifest file to access its properties and returns the manifest object. This can be useful for scripts that need to dynamically discover and utilize module metadata, such as version, dependencies, and exported functions.
+
+.PARAMETERS
+This function does not accept any parameters.
+
+.EXAMPLE
+$manifest = Get-ModuleManifest
+This example calls the `Get-ModuleManifest` function to retrieve the module manifest object and stores it in the variable `$manifest`.
+
+#>
+function Get-ModuleManifest {
```

Comment:
Needs to be renamed to `Get-PodeModuleManifest`, and probably more suited to being in `Private/Helpers.ps1` as it's just a helper function rather than anything related to versioning